types/filter.d.ts

```diff
@@ -7,8 +7,13 @@ export { FilterType } from "../src/filter";
 
 export declare type SingleValue = string | number | boolean | Date;
 export declare type RangeDataType = number | Date;
+export declare type Param = {
+    value: string;
```

Comment:
>value: string; [](start = 4, length = 14)

Can parameters really be string only?

Reply:
fair point, it's flexible enough to take other values. The udf does some basic conversion. I'll update this to a singleValue

---
In reply to: [260920584](https://github.com/ssivov/filter-conditions/pull/14#discussion_r260920584) [](ancestors = 260920584)